client/packages/invoices/src/Returns/OutboundDetailView/DetailView.tsx

```diff
@@ -114,3 +111,19 @@ export const OutboundReturnsDetailView: FC = () => {
     </React.Suspense>
   );
 };
+
+export const getNextItemId = (
+  lines: { itemId: string }[],
+  currentItemId: string | null
+) => {
+  if (!lines || !currentItemId) return undefined;
+  const currentItemIndex = lines.findIndex(
+    line => line.itemId === currentItemId
+  );
+  if (currentItemIndex === -1) return;
+
+  const nextItemIndex = lines.findIndex(
+    (line, index) => index > currentItemIndex && line.itemId !== currentItemId
+  );
+  return nextItemIndex === -1 ? undefined : lines[nextItemIndex]?.itemId;
+};
```

Comment:
Exported and re-used in Inbounds

Reply:
Could we put this in packages/invoices/src/utils.ts?

Reply:
Yup, good call: 5c125e06c